test/services/xccdf_report_migration_test.rb

```diff
@@ -49,7 +49,7 @@ class XCCDFReportMigrationTest < ActiveSupport::TestCase
     conflict_profile = Profile.create!(
       account: accounts(:test),
       hosts: [hosts(:one)],
-      name: 'footitle',
+      name: 'footitle2',
```

Comment:
I needed to change the title as this conflicting profile couldn't occur anymore due to the uniqueness constraint. The migration test however is still valid as all we compare here is the ref IDs of `conflict_profile` and `@profile`